### PR TITLE
x86: add experimental ps2 emulator

### PR DESCRIPTION
**Pull request by gizmo98**
Lazy implementation. Just use Ubuntu 16.04 pcsx2 package.

### CODE REVIEW

File: scriptmodules/emulators/pcsx2.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+rp_module_id="pcsx2"
+rp_module_desc="PS2 emulator PCSX2"
+rp_module_help="ROM Extensions: .bin .iso .img .mdf .z .z2 .bz2 .cso .ima .gz\n\nCopy your PS2 roms to $romdir/ps2"
+rp_module_section="exp"
+rp_module_flags="!arm"
+
+function sources_pcsx2() {
+    getDepends pcsx2
+}
+
```
Comment (joolswills) [inline scriptmodules/emulators/pcsx2.sh]: should be aptInstall under install_bin

**Comment by gizmo98** [comment]
All right. Fixed it.

**Comment by joolswills** [comment]
Thanks!

**Comment by HerbFargus** [comment]
already in the wiki too :P https://github.com/retropie/retropie-setup/wiki/Playstation-2

**Pull request merged**